Automattic/jetpack
# Carousel refactor: use svgs for prev/next instead of default swiper font icons

Fixes: https://github.com/Automattic/jetpack/pull/20107#discussion_r660519441

#### Changes proposed in this Pull Request:

* Switch out the default swiper prev/next font icons for SVGs 

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Check out this PR locallly
* Turn on Jetpack carousel option and add a gallery
* In front end make sure carousel loads and SVGs are used for prev/next icons, and that hover states work as expected
